public/style.css

```diff
@@ -0,0 +1,82 @@
+:root {
+	--primary-colour: #CCC;
+	--secondary-colour: #555;
+	--link-colour: #000;
+	--link-colour-alt: #FFDB3A;
```

Comment:
nice use of css vars :)

Reply:
I'm starting to appreciate CSS variables a little bit more now, especially for base stuff like colours and typography!